src/verify.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+
+exercises = [
+    ("contracts/beginner", ["ex00", "ex01", "ex02", "ex03", "ex04"]),
+    ("contracts/advanced", ["ex00", "ex01"])
+]
+
+
+def is_exercise_done(path: str) -> bool:
```

Comment:
I would find it more consistent with the rest of the codebase if we stick to having all the logic in classes. Just encapsulate these two functions in an `ExerciseSeeker` class for example.

Reply:
Done